samples/nrf9160/modem_shell/README.rst

```diff
@@ -933,6 +933,63 @@ For example:
 
    west build -p -b nrf9160dk_nrf9160_ns -- -DOVERLAY_CONFIG=overlay-non-offloading.conf
 
+BT shell support
+================
+
+To build the MoSh sample with Zephyr BT shell command support, use ``-DDTC_OVERLAY_FILE=bt.overlay`` and  ``-DOVERLAY_CONFIG=overlay-bt.conf`` options.
+When running this configuration, BT scanning and advertising can be made with ``bt`` command. Compilation for an example:
+
+.. code-block:: console
+
+   west build -p -b nrf9160dk_nrf9160_ns -- -DDTC_OVERLAY_FILE="bt.overlay" -DOVERLAY_CONFIG="overlay-bt.conf"
+
+Additionally, nRF52840 side of the development kit needs to be programmed as instructed in :ref:`lte_sensor_gateway`. Compilation of :ref:`bluetooth-hci-lpuart-sample`:
```

Comment:
```suggestion
Additionally, you need to program the nRF52840 DK as instructed in :ref:`lte_sensor_gateway`.

Compile the :ref:`bluetooth-hci-lpuart-sample` as follows:
```

Reply:
Thanks for the comments. But this one is not quite correct. I'm referring to nRF52840 that is on/part of the nRF9160 DK. I.e. it is not nRF52840 DK.

Reply:
Thought this might go wrong. But you got the idea how to fix this.

Reply:
Yes, I think so. All has been fixed now.